src/Pages/EmailList/email-list.css

```diff
@@ -0,0 +1,45 @@
+.email-list {
+    background-color: #ecebeb;
+    height: 100vh;
+}
+
+#email-filter-button {
+    background-color: rgb(2, 107, 126) !important;
```

Comment:
do you need important?

Reply:
React-strap force you to mark colors as ```important``` if you want to use any other color beside the default list.
@realCalvin you can try the [deafult colors](https://reactstrap.github.io/components/buttons/) if it looks nice on our website

Reply:
also that color look like the default ```color = info```; maybe try that

Reply:
removed the importants